test/fixtures.json

```diff
@@ -1,16 +1,37 @@
 {
   "alphabets": {
     "base2": "01",
+    "base16": "0123456789ABCDEF",
```

Comment:
Can you change this to `0123456789abcdef`?  It'll be consistent with node `Buffer`s then.